base/applications/rapps/lang/en-US.rc

```diff
@@ -191,9 +192,6 @@ BEGIN
     IDS_APPTITLE "ReactOS Applications Manager"
     IDS_APP_AUTHORS "Copyright 2009 Dmitry Chapyshev"
     IDS_SEARCH_TEXT "Search…"
-    IDS_INSTALL "Install"
-    IDS_UNINSTALL "Uninstall"
-    IDS_MODIFY "Modify"
```

Comment:
What happened to those strings?

Reply:
They only used by `CMainToolbar::Create` and duplicated by `IDS_TOOLTIP_*`

Reply:
I see... Well, in case someone wants to change the tooltip text, then having 2 different strings would matter.